seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -1143,7 +1143,7 @@
         }
       ],
       "description": [
-        "<code>Google Fonts</code> is a great way to find free fonts to use in a project.",
+        "<code><a href='https://fonts.google.com/' target='_blank'>Google Fonts</a></code> is a great way to find free fonts to use in a project.",
```

Comment:
Yeah that was something I was wondering about. Should I just take the link out for now?